modules/frontend/frontend.go

```diff
@@ -0,0 +1,227 @@
+package frontend
+
+import (
+	"context"
+	"encoding/binary"
+	"fmt"
+	"github.com/cortexproject/cortex/pkg/querier/frontend"
+	"github.com/cortexproject/cortex/pkg/util"
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+	"github.com/weaveworks/common/user"
+	"net/http"
+)
+
+type FrontendConfig struct {
+	frontend.Config `yaml:",inline"`
+	ShardNum int    `yaml:"shard_num,omitempty"`
+}
+
+func (cfg *FrontendConfig) ApplyDefaults() {
+	cfg.Config.CompressResponses = false
```

Comment:
i'm curious why we would default this to false.  it seems like you were just following suit with Cortex, but unless I'm misunderstanding the functionality it feels like this should be true.